rover-apps/common/include/WatchdogModule.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "Module.h"
+#include "mbed.h"
+
+class WatchdogModule final : public Module {
+ public:
+  /* Initiates the watchdog with a countdown
+   * @param countdown - max timeout of the watchdog
+   * */
+  WatchdogModule(std::chrono::milliseconds countdown = 5000ms);
+
+  /* Logs the last reset due to the watchdog
+   * */
+  void Watchdog_logLastResetReason(void);
```

Comment:
Can you remove this function and just call `WatchdogWrapper::logResetReason()` inside the constructor?